cellxgene_resources/cellxgene_mods.py

```diff
@@ -384,6 +387,13 @@ def evaluate_obs_schema(obs, labels=False):
         for o in portal_obs_fields:
             if o in obs.keys():
                 report(f'schema conflict - {o} in obs\n', 'ERROR')
+    if 'cell_type_ontology_term_id' in obs.columns\
+            and 'unknown' in obs['cell_type_ontology_term_id'].unique()\
+            and len([i for i in obs['assay_ontology_term_id'].unique() if i in ['EFO:0022860','EFO:0022859','EFO:0022857']])==0:
```

Comment:
instead of dropping Visium Datasets from the check, why not drop in_tissue:0 obs then do the same check?
could just check if in_tissue in obs.columns to avoid future update of EFO terms

Reply:
Good idea! Update for script to check visium using in_tissue rather than efo and subset to in_tissue:1 for cell_type unknown check